docs/nebula-design-concepts.md

```diff
@@ -0,0 +1,216 @@
+# Data modeling and design concept
+------
+**Nebula Graph** is comprised of four components: storage layer, metadata service, compute layer and client console. In this documentation, we'll discuss data modeling and architecture of Nebula Graph.
+
+---
+## Directed property graph
+
+Modeling data with directed property graph, Nebula Graph is very easy to understand.
+That is to say, Nebula Graph consists of two graph elements: vertex and edge. Each
+vertex represents an entity, and each edge represents how two vertices are associated.
+
+![image][example1]
+
+<!-- ### Vertex
+
+Vertices are typically used to store entity information. In the preceding example, vertices are **players** and **team**.
```

Comment:
to store entities with the properties.

teams

Reply:
Please note contents commented out here, I introduced graph elements in my way.

Reply:
vertices are **player** and **team**